project/src/components/Layout/Header/Nav/DesktopNav.tsx

```
@@ -14,11 +14,20 @@
 
 type ItemProps = NavItem & {
     currentPath: string;
+    index: number;
 };
 
-function NavItem({ href, label, delay, currentPath }: ItemProps) {
+function NavItem({ href, label, currentPath, index }: ItemProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'NavItem'. Consider to delete it or rename it.

'NavItem' is defined here:


(lint/suspicious/noRedeclare)